lib/postgrex_wal/messages/helper.ex

```diff
@@ -0,0 +1,58 @@
+defmodule PostgrexWal.Messages.Helper do
```

Comment:
helper 我们一般用到 view 里面或者测试里面比较多

```suggestion
defmodule PostgrexWal.Messages.Util do
```

Reply:
已修改。